WordPress/Classes/Networking/SharingServiceRemote.swift

```diff
@@ -0,0 +1,297 @@
+                    conn.service = dict.stringForKey(ConnectionDictionaryKeys.service) ?? conn.service
+                    conn.type = dict.stringForKey(ConnectionDictionaryKeys.type) ?? conn.type
+                    conn.userID = dict.numberForKey(ConnectionDictionaryKeys.userID) ?? conn.userID
+
+                    return conn
+                }
+
+                onSuccess(keyringConnections)
+            },
+            failure: { (operation: AFHTTPRequestOperation!, error: NSError!) -> Void in
+                failure?(error)
+        })
+    }
+
+
+    /**
+     *  @brief      Fetches the current user's list of Publicize connections for the specified site's ID.
+     *
+     *  @param      siteID      The WordPress.com ID of the site.
+     *  @param      success     An optional success block accepting an array of `RemotePublicizeConnection` objects.
+     *  @param      failure     An optional failure block accepting an `NSError` argument.
+     */
+    public func getPublicizeConnections(siteID:NSNumber, success: ([RemotePublicizeConnection] -> Void)?, failure: (NSError! -> Void)?) {
+        let endpoint = "sites/\(siteID)/publicize-connections"
+        let path = self.pathForEndpoint(endpoint, withVersion: ServiceRemoteRESTApiVersion_1_1)
+
+        api.GET(path,
+            parameters: nil,
+            success: {(operation:AFHTTPRequestOperation!, response:AnyObject!) -> Void in
+                guard let onSuccess = success else {
+                    return
+                }
+
+                let responseDict = response as! NSDictionary
+                let connections:Array = responseDict.arrayForKey(ConnectionDictionaryKeys.connections)
+                let publicizeConnections:[RemotePublicizeConnection] = connections.map { (let dict) -> RemotePublicizeConnection in
+                    let conn = self.remotePublicizeConnectionFromDictionary(dict as! NSDictionary)
+                    return conn
+                }
+
+                onSuccess(publicizeConnections)
+            },
+            failure: { (operation: AFHTTPRequestOperation!, error: NSError!) -> Void in
+                failure?(error)
+        })
+    }
+
+
+    /**
+     *  @brief      Create a new Publicize connection bweteen the specified blog and
+     *              the third-pary service represented by the keyring.
+     *
+     *  @param      siteID                  The WordPress.com ID of the site.
+     *  @param      keyringConnectionID     The ID of the third-party site's keyring connection.
+     *  @param      success     An optional success block accepting a `RemotePublicizeConnection` object.
+     *  @param      failure     An optional failure block accepting an `NSError` argument.
+     */
+    public func createPublicizeConnection(siteID:NSNumber,
+        keyringConnectionID:NSNumber,
+        externalUserID:String?,
+        success: (RemotePublicizeConnection -> Void)?, failure: (NSError! -> Void)?) {
```

Comment:
I waffled on this tbh.  I could go either way. I think the extra indentation helps some but putting the { on its own line might help more.